### PR TITLE
release v1.5.3

### PR DESCRIPTION
**Pull request by calebcartwright**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,34 +41,7 @@
 
 ### Misc
 
-- Prevent ICE when parsing invalid attributes in `cfg_if!` macros [#5728](https://github.com/rust-lang/rustfmt/issues/5728)
```
Comment (calebcartwright) [inline CHANGELOG.md:37]: Moved up to FIXED section, as this was a bug producing an ice that was fixed vs. a miscellaneous change

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,34 +41,7 @@
...
-
-### Misc
-
-- rustfmt now internally supports the ability to have both stable and unstable variants of a configuration option [#5378](https://github.com/rust-lang/rustfmt/issues/5378). This ability will allow the rustfmt team to make certain configuration options available on stable toolchains more quickly because we no longer have to wait for _every_ variant to be stable-ready before stabilizing _any_ variant. 
-
-### Install/Download Options
-- **rustup (nightly)** - nightly-2023-01-24
-- **GitHub Release Binaries** - [Release v1.5.2](https://github.com/rust-lang/rustfmt/releases/tag/v1.5.2)
-- **Build from source** - [Tag v1.5.2](https://github.com/rust-lang/rustfmt/tree/v1.5.2), see instructions for how to [install rustfmt from source][install-from-source]
```
Comment (calebcartwright) [inline CHANGELOG.md:64]: ugly diff from the merge conflict resolution the sync, but this section is just fully duplicated

File: CHANGELOG.md [hunk 3]
```diff
@@ -34,34 +41,7 @@
...
+- Update various dependencies, including `syn`, `cargo_metadata`, `env_logger`, and `toml`
```
Comment (calebcartwright) [inline CHANGELOG.md:45]: install/download options section is missing from here, intentionally. at this point in the process we no longer know what any of these values are going to be, and i'm tired of taking a semi-random guess at it only to have to update later

**Review by ytmimi — APPROVED** [review]
Appreciate all the explanation  👍🏼

**Comment by calebcartwright** [comment]
these are essentially moot at this point given the shift of all the rustc crates being shipped rmeta files (the rustfmt binaries produced by these aren't actually really usable anymore) but :crossed_fingers: it passes anyway

https://github.com/rust-lang/rustfmt/actions/runs/5318345881

**Pull request merged**